src/nvim/eval/funcs.c

```diff
@@ -4514,8 +4516,8 @@ static void f_has(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 
   bool n = false;
   const char *const name = tv_get_string(&argvars[0]);
-  for (size_t i = 0; i < ARRAY_SIZE(has_list); i++) {
-    if (STRICMP(name, has_list[i]) == 0) {
+  for (size_t i = 0; i < ARRAY_SIZE(kHasList); i++) {
```

Comment:
Unless I hear other core contributors in favor of this, I do not think all of these endless name changes are worth the entropy.

Closing any PRs with these changes.

Reply:
I don't understand the name changes either. What does the `k` mean exactly?

I do think that some of the other changes in this PR are valuable though, namely wrapping macro function arguments in parens.

Reply:
> I do think that some of the other changes in this PR are valuable though, namely wrapping macro function arguments in parens.

Yes. But these PRs should not include name changes, I've mentioned this in previous PRs...

Reply:
> I don't understand the name changes either. What does the `k` mean exactly?
> 
> I do think that some of the other changes in this PR are valuable though, namely wrapping macro function arguments in parens.

I didn't come up with any of the rules btw: https://neovim.io/doc/user/dev_style.html#dev-style

> Yes. But these PRs should not include name changes, I've mentioned this in previous PRs...

I must've missed your comment regarding the name changes, maybe it was in the other PR which I've been working on in parallel. Sorry about that.